eng/pipelines/aggregate-reports.yml

```diff
@@ -110,21 +80,54 @@ jobs:
         Copy-Item eng/code-quality-reports/src/main/resources/index.html $(Build.ArtifactStagingDirectory)
         Copy-Item eng/code-quality-reports/src/main/resources/dependency-allowlist.html $(Build.ArtifactStagingDirectory)/staging
 
-        # What is this?
-        # copy output/pom.client.html (Join-Path $(Build.ArtifactStagingDirectory) "staging")
-
         Copy-Item eng/common/InterdependencyGraph.html $(Build.ArtifactStagingDirectory)/staging
         Copy-Item data.js $(Build.ArtifactStagingDirectory)/staging
       displayName: 'Copy reports to artifact staging'
 
+    - publish: $(Build.ArtifactStagingDirectory)
+      condition: succeededOrFailed()
+      displayName: 'Publish Report Artifacts'
+      artifact: reports
+  - job: Compliance
+    displayName: Analyze (No build)
+    timeoutInMinutes: 120
+    variables:
+      - template: templates/variables/globals.yml
+    pool:
+      vmImage: 'ubuntu-18.04'
```

Comment:
Cred scan is asking win os to run their tool. Better to double check whether cred scan is able to run under ubuntu

Reply:
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=741047&view=logs&j=38d918fc-df5a-5870-2b80-6c17fdb13baf&t=1c955c51-9d5f-5f71-c490-e8dcdbd43fdf

It seems run into some errors. Can we switch back to win vm and test?